File: plsql/CSharp/PlSqlBaseLexer.cs
using Antlr4.Runtime;

public abstract class PlSqlBaseLexer : Lexer
{
    public PlSqlBaseLexer(ICharStream input)
        : base(input)
    {
    }

    protected bool IsNewlineAtPos(int pos)
    {
        int la = _input.La(pos);
        return la == -1 || la == '\n';
    }
}

File: plsql/Java/PlSqlBaseLexer.java
import org.antlr.v4.runtime.*;

public abstract class PlSqlBaseLexer extends Lexer
{
    public PlSqlBaseLexer(CharStream input) 
	{
		super(input);
    }

    protected boolean IsNewlineAtPos(int pos)
    {
        int la = _input.LA(pos);
        return la == -1 || la == '\n';
    }
}
